script/gen-filenames.js

```diff
@@ -73,7 +73,7 @@ const main = async () => {
       // Ignore empty lines
       .filter(line => line)
       // Get the relative path
-      .map(line => path.relative(rootPath, line))
+      .map(line => path.relative(rootPath, line).replace(/\\/g, '/'))
```

Comment:
```suggestion
      .map(line => path.posix.relative(rootPath, line))
```

Reply:
https://nodejs.org/api/path.html#path_windows_vs_posix

Reply:
I tried that first, it didn't work and returned empty. I thought `path.posix.relative` means "the given path is posix, resolve that and give me the result" and so in this case the result was empty.
I can check the reason again.

Reply:
ah, ok. it seems to me that there ought to be a more rigorous way of handling paths but fine, lgtm for now, i suppose that if we put `\\` or `/` in any file names in our repo then we deserve what we get.